server/views/temporary-accommodation/bedspace-search/results.njk

```diff
@@ -0,0 +1,143 @@
+{% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/details/macro.njk" import govukDetails %}
+
+{% from "moj/components/filter/macro.njk" import mojFilter %}
+{% from "../../partials/breadCrumb.njk" import breadCrumb %}
+{% from "../../components/place-context-header/macro.njk" import placeContextHeader %}
+
+{% extends "../../partials/layout.njk" %}
+
+{% set pageTitle = "Bedspace search results (" + results.results.length + ") - " + applicationName %}
+{% set mainClasses = "app-container govuk-body" %}
+
+{% block beforeContent %}
+    {{ breadCrumb('Search for available bedspaces', []) }}
+    {{ placeContextHeader(placeContext) }}
+{% endblock %}
+
+{% block content %}
+    {% include "../../_messages.njk" %}
+
+    <h1 class="govuk-heading-l">Bedspace search results ({{ results.results.length }})</h1>
+
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-one-third govuk-!-margin-bottom-8">
+            {% set filterOptionsHtml %}
+                {% include './partials/search-fields.njk' %}
+
+                <div class="govuk-button-group">
+                    {{ govukButton({
+                        text: "Apply filters",
+                        preventDoubleClick: true
+                    }) }}
+
+                    <a href="{{ paths.bedspaces.search({}) }}" class="govuk-link">Clear filters</a>
+                </div>
+            {% endset %}
+
+            <form action="{{ paths.bedspaces.search({}) }}" method="get">
+                {{ mojFilter({
+                    heading: {
+                        text: 'Filter'
+                    },
+                    optionsHtml: filterOptionsHtml
+                }) }}
+            </form>
+        </div>
+
+        <div class="govuk-grid-column-two-thirds">
+            {% if results.results.length === 0 %}
+                <p class="govuk-body-l">
+                    There are no available bedspaces
+                    {{ parseNaturalNumber(durationDays) }} {{ 'day' if parseNaturalNumber(durationDays) === 1 else 'days' }}
+                    from {{ formatDate(startDate) }}.
+                </p>
+                <p class="govuk-body-l">Try changing one or more of your search criteria to get more results.</p>
+            {% else %}
+                {% for result in results.results %}
```

Comment:
🤔 I'm wondering if we can improve this as its tricky dealing with `results.results`. Its tricky to work out what exactly is results of results by looking only at the view